### PR TITLE
feat: new pagination

### PR DESCRIPTION
**Pull request by Harris-YWang**
## Description

As per asc-32, create a new pagination component with react only, and remove the dependency on the `react-bootstrap` https://github.com/Adslot/adslot-ui/pull/963

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

**Comment by lightbringer1991** [comment]
have we gotten some design review yet?

**Review by Harris-YWang — COMMENTED** [review]
design reviewed by @pedroscoulloupas, the new look of this component is 
Kapture 2020-10-26 at 16 53 35

File: src/styles/icons/point-left.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path fill="none" d="M0 0h16v16H0z"/><path fill="none" d="M2 2h12v12H2z"/><path d="M4.5 8c0 .2.1.5.2.6l5 5c.4.4 1 .4 1.4 0 .4-.4.4-1 0-1.4L6.8 8l4.4-4.3c.4-.3.4-1 0-1.4s-1-.4-1.4 0l-5 5.1c-.2.2-.3.4-.3.6z" fill="#838383"/></svg>
```
Comment (Harris-YWang) [inline src/styles/icons/point-left.svg:1]: `svgo` compress tool requires no eof for `svg` files in adslot-ui

File: src/styles/icons/point-right.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path fill="none" d="M0 0h16v16H0z"/><path fill="none" d="M2 2h12v12H2z"/><path d="M11.5 8c0-.2-.1-.5-.2-.6l-5-5c-.4-.4-1-.4-1.4 0s-.4 1 0 1.4L9.2 8l-4.4 4.3c-.4.3-.4 1 0 1.4s1 .4 1.4 0l5-5.1c.2-.2.3-.4.3-.6z" fill="#838383"/></svg>
```
Comment (Harris-YWang) [inline src/styles/icons/point-right.svg:1]: same as above

File: src/components/Pagination/index.jsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Button from '../Button';
+import './styles.scss';
+
+const Pagination = ({ className, activePage, pageCount, onSelect, prev, next }) => (
+  <div data-testid="pagination-wrapper" className={classnames('aui--pagination', className)}>
+    {activePage !== 1 && prev && (
+      <Button
+        key="pagination-previous"
```
Comment (lightbringer1991) [inline src/components/Pagination/index.jsx:11]: I guess we don't need `key` prop here?

**Comment by lightbringer1991** [comment]
In case there are 5 pages, seems a bit weird
Oct-26-2020 17-45-30
also should we disable the active page?

File: src/components/Pagination/index.jsx (hunk 3 above)
Comment (Harris-YWang) [reply src/components/Pagination/index.jsx]: fixed

**Comment by Harris-YWang** [comment]
> In case there are 5 pages, seems a bit weird
> Oct-26-2020 17-45-30
> also should we disable the active page?

Checked with Pedros, now when the page count is 5, the new look will be 
Kapture 2020-10-30 at 12 16 32

**Review by lteacher — APPROVED** [review]

**Review by lightbringer1991 — APPROVED** [review]
LGTM

**Review by superpower1 — APPROVED** [review]

**Pull request merged**